### ISSUE TITLE
building other examples

### ISSUE THREAD

**Issue by BarzinM** [ethz-adrl/control-toolbox#183]
Hi, sorry if this question is too basic. How can I build other ct_optcon examples such as `control-toolbox/ct_optcon/examples/NLOC_MPC.cpp`?
When I build the project, it automatically builds `ex_NLOC_boxConstrained` and `ex_NLOC_generalConstrained` but other examples files such as NLOC_MPC don't get built.
I'm *not* using catkin system.

**Comment by markusgft** [comment]
Hi, probably you need to build with `-DBUILD_EXAMPLES=true` !

**Comment by justinberi** [comment]
Hi,
I was following the [QuickStart](https://github.com/ethz-adrl/control-toolbox/wiki/Quickstart) instructions at head e7a29a6 and using catkin tools and running with `-DBUILD_EXAMPLES=true`.
For me `ex_NLOC_boxConstrained` and `ex_NLOC` are not built. 

For `ex_NLOC_boxConstrained` not building.
HPIPM was not installed automagically. I had to run `ct/install_hpimp.sh`.
Running that script fixed the problem.

For `ex_NLOC` not building.
I noticed this executable (and more) are commented out in `ct_optcon/examples/CMakeLists.txt`.
Adding them back fixed that problem.
